server/utils/security/signature_match.py

```
@@ -100,6 +104,24 @@ def _r(pattern: str, technique: str, rule_id: str, desc: str):
 _r(r"\bchmod\s+[0-7]*[2367][0-7]*\s+/", T_ABUSE_ELEVATION, "privesc-chmod-suid", "Setting SUID/SGID on system files")
 
 
+# --- SigmaHQ-sourced rules (loaded from vendored YAML at import time) ---
+logger = logging.getLogger(__name__)
+
+def _load_sigma():
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `_load_sigma`

Add return type annotation: `None`

(ANN202)